modules/library/src/main/scala/zio/elasticsearch/aggregation/Aggregations.scala

```diff
@@ -68,12 +68,12 @@ sealed trait TermsAggregation
 private[elasticsearch] final case class Terms(
   name: String,
   field: String,
-  order: Set[AggregationOrder],
+  order: List[AggregationOrder],
   subAggregations: List[SingleElasticAggregation],
   size: Option[Int]
 ) extends TermsAggregation { self =>
   def orderBy(order: AggregationOrder, orders: AggregationOrder*): TermsAggregation =
-    self.copy(order = self.order + order ++ orders.toSet)
+    self.copy(order = (self.order :+ order) ++ orders)
```

Comment:
Same as above:
- self.order is copied twice this way instead of just once
- should we use chunk?